common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/EstsTelemetry.java

```diff
@@ -81,6 +86,27 @@ public void setupLastRequestTelemetryCache(@NonNull final Context context) {
         }
     }
 
+    /**
+     * Creates an entry for a Current Telemetry object for the passed in command based on whether
+     * the command is eligible for telemetry. Saves the telemetry object to telemetry map.
+     *
+     * @param command The command for which to capture telemetry
+     */
+    public void initTelemetryForCommand(@NonNull final BaseCommand command) {
+        setupLastRequestTelemetryCache(command.getParameters().getAppContext());
+        final String correlationId = command.getParameters().getCorrelationId();
+        if (command.isEligibleForEstsTelemetry()) {
+            final CurrentRequestTelemetry currentRequestTelemetry = new CurrentRequestTelemetry();
+            mTelemetryMap.put(correlationId, currentRequestTelemetry);
+        }
+    }
+
+    private boolean isDisabled() {
```

Comment:
When would we disable this telemetry? If this method is only to check if we have telemetry or not, we might to have rename this method more appropriately

Reply:
It is disabled in the sense disabled for a command such as the GetDeviceModeCommand that returns false for `isEligibleForEstsTelemetry`, but I agree this can be renamed to something better. Perhaps `isCurrentTelemetryAvailable` or `isTelemetryCaptured`?

Reply:
Yup, `isCurrentTelemetryAvailable` seems better to me

Reply:
I'd suggest using 'eligible' instead of 'avaiable' - so that we know it came from isEligibleForEstsTelemetry

Reply:
That may not be necessarily true. If we refactor the code in the future and introduce a bug then we may not have telemetry due to a bug and not simply because a command wasn't eligible.

Reply:
Addressed